### DISCUSSION TITLE
Date pickers: add shortcuts

### DISCUSSION THREAD

**Discussion by souhagaa** [mantinedev/mantine#6832] [Feature requests]
It would be nice to add date shortcuts to date pickers these are super helpful to the users

![image](https://github.com/user-attachments/assets/5f10b1ef-1aaf-4b81-add5-74582622331e)

**Comment by Vagizova** [comment]
Is there a way to do this? I still haven't found one.

**Reply by souhagaa** [reply]
In my case, I am migrating from BlueprintJS to Mantine, since Blueprint has the shortcuts and the date pickers are independent packages in the newer versions I just used those
